### ISSUE TITLE
Unnamed classes

### ISSUE THREAD

**Issue by harg** [pangloss/vim-javascript#401]
Unamed classes are not properly handled. See the screenshot below :

![js_unnamed_class](https://cloud.githubusercontent.com/assets/34398/14814952/f7439e94-0ba8-11e6-84ca-0246081bcb0f.png)

https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Classes

BTW thanks for your work on this syntax definition!

**Comment by megapctr** [comment]
why not just declare the class like this? `class Polygon {...}`

**Comment by harg** [comment]
I'm playing with es2015 classes and I noticed it...
I'm not aware of any usecases for this right now... maybe anonymous inner classes "ala" Java...

```javascript
class c {
  constructor() {
    this.innerClass = new class {
      constructor() {
        console.log('hello innerClass')
      }
    }
  }
};

const instance = new c
instance.innerClass // output: hello innerClass
```

**Comment by amadeus** [comment]
@megapctr @harg I agree that it's perhaps a bit weird to define a class without a name, but we should still support it for sure, since it's valid syntax

**Comment by amadeus** [comment]
It's a pretty simple fix, see #402

**Comment by amadeus** [comment]
Should be fixed in the `develop` branch
